SpEC/com_motion.py

```diff
@@ -208,10 +208,8 @@ def remove_avg_com_motion(path_to_waveform_h5='rhOverM_Asymptotic_GeometricUnits
     # Read the waveform data in
     w_m = read_from_h5(path_to_waveform_h5)
 
-    # Insert '_CoM' into the original `path_to_waveform_h5`; this will be the output path
-    path_to_new_waveform_h5 = path_to_waveform_h5.split('.h5', 1)
-    path_to_new_waveform_h5.insert(1, '_CoM.h5')
```

Comment:
With this explanation, the original code makes sense.  However, to improve code-clarity, it is useful to make assumptions explicit.  So, instead of adding an opaque ",1" in an already
opaque replace statement, I added a check that there is only **one** .h5 within the entire path; if not an error is printed.  

NOTE: I don't know the error reporting strategy of scri, so this is a (likely) unacceptable print statement, followed with sys.os.exit().